longshot/application/main.tf

```diff
@@ -0,0 +1,218 @@
+variable "pipeline" {}
+variable "pipeline_stage" {}
+variable "name" {}
+variable "host" {}
+
+variable "web_size" {
+  default = "Standard-1X"
+}
+
+variable "web_scale" {
+  default = "1"
+}
+
+variable "queue_size" {
+  default = "Standard-1X"
+}
+
+variable "queue_scale" {
+  default = "1"
+}
+
+variable "redis_type" {
+  default = "hobby-dev"
+}
+
+variable "database_name" {}
+
+variable "database_type" {
+  default = "db.t2.medium"
+}
+
+variable "database_size" {
+  default     = 100
+  description = "The amount of storage to allocate to the database, in GB."
+}
+
+variable "email_name" {}
+variable "email_address" {}
+
+variable "papertrail_destination" {}
+
+data "aws_ssm_parameter" "database_username" {
+  name = "/${var.name}/rds/username"
+}
+
+data "aws_ssm_parameter" "database_password" {
+  name = "/${var.name}/rds/password"
+}
+
+data "aws_ssm_parameter" "mandrill_api_key" {
+  name = "/mandrill/api-key"
+}
+
+resource "heroku_app" "app" {
+  name   = "dosomething-${var.name}"
+  region = "us"
```

Comment:
region should be a parameter for template

Reply:
I don't think we'll be creating applications in non-US regions anytime soon (even when we re-open global traffic, we'll likely let Fastly handle local POPs for us), so this seems safe to leave as a hard-coded string to me.